src/more-infos/more-info-media_player.html

```diff
@@ -64,6 +64,16 @@
           on-change='volumeSliderChanged' class='flex'>
         </paper-slider>
       </div>
+      <div class='controls layout horizontal justified' hidden$='[[!supportsSelectSource]]'>
+          <paper-icon-button icon="mdi:login-variant"></paper-icon-button>
+          <paper-dropdown-menu style="margin-left: 10px;" no-label-float selected-item-label='{{selectedSource}}' on-tap='stopPropagation'>
```

Comment:
Why did you add the `on-tap` event listener ? I would expect this to be handled by the `paper-dropdown` ?